gt/main.c

```diff
@@ -749,6 +900,9 @@ gt_proc(void *arg)
 	uint16_t rx_queue = instance->rx_queue;
 	uint16_t tx_queue = instance->tx_queue;
 
+	/* The mbuf death row contains packets to be freed. */
+	struct rte_ip_frag_death_row death_row;
```

Comment:
1. Variable `death_row` is first used without initialization in `gt_proc()` to call `gt_reassemble_incoming_pkt()`. It's my understanding that we must have `death_row.cnt = 0;` before doing that.

2. Variable `death_row` is only used inside of the `if` statement `if (pkt_info.frag) {` later in the code, so we should move it to there.